terraform/development/shared/main.tf

```diff
@@ -0,0 +1,168 @@
+    encrypt        = true
+    key            = "core-development.tfstate"
+    region         = "eu-west-2"
+    role_arn       = "arn:aws:iam::815624722760:role/developer"
+  }
+}
+
+provider "aws" {
+  region = "eu-west-2"
+
+  assume_role {
+    role_arn = local.provider_role_arn
+  }
+}
+
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+
+  assume_role {
+    role_arn = local.provider_role_arn
+  }
+}
+
+locals {
+  prefix = "core-dev"
+
+  app_host                  = "review.submit-social-housing-data.levellingup.gov.uk"
+  load_balancer_domain_name = "review.lb.submit-social-housing-data.levellingup.gov.uk"
+
+  provider_role_arn = "arn:aws:iam::837698168072:role/developer"
+
+  enable_aws_shield = false
+
+  application_port = 8080
+  database_port    = 5432
+  redis_port       = 6379
+}
+
+module "application_roles" {
+  source = "../../modules/application_roles"
+
+  github_actions_role_arn = "arn:aws:iam::815624722760:role/core-application-repo"
+
+  prefix                               = local.prefix
+  bulk_upload_bucket_access_policy_arn = module.bulk_upload.read_write_policy_arn
+  database_data_access_policy_arn      = module.database.rds_data_access_policy_arn
+  export_bucket_access_policy_arn      = module.cds_export.read_write_policy_arn
+
+  secret_arns = [
+    module.application_secrets.api_key_secret_arn,
+    module.application_secrets.govuk_notify_api_key_secret_arn,
+    module.application_secrets.os_data_key_secret_arn,
+    module.application_secrets.rails_master_key_secret_arn,
+    module.application_secrets.sentry_dsn_secret_arn
+  ]
+}
+
+module "application_secrets" {
+  source = "../../modules/application_secrets"
+}
+
+module "application_security_group" {
+  source = "../../modules/application_security_group"
+
+  prefix                          = local.prefix
+  application_port                = local.application_port
+  database_port                   = local.database_port
+  db_security_group_id            = module.database.rds_security_group_id
+  load_balancer_security_group_id = module.front_door.load_balancer_security_group_id
+  redis_port                      = local.redis_port
+  vpc_id                          = module.networking.vpc_id
+}
+
+module "bulk_upload" {
+  source = "../../modules/bulk_upload"
+
+  prefix = local.prefix
+}
+
+module "cds_export" {
+  source = "../../modules/cds_export"
+
+  prefix = local.prefix
+}
+
+module "certificates" {
+  source = "../../modules/certificates"
+
+  providers = {
+    aws.us-east-1 = aws.us-east-1
+  }
+
+  cloudfront_domain_name    = local.app_host
+  load_balancer_domain_name = local.load_balancer_domain_name
+}
+
+module "database" {
+  source = "../../modules/rds"
+
+  allocated_storage       = 100
+  backup_retention_period = 0 # no backups
+
+  apply_changes_immediately          = true
+  enable_primary_deletion_protection = false
+  highly_available                   = false
+  skip_final_snapshot                = true

```

Comment:
For dev / review-apps I don't think we need high availability or a final snapshot on deletion. Thinking over I would enable deletion protection though, because we only have the one cluster not several for each review app so will update this

Reply:
Enabled deletion protection 👍